### DISCUSSION TITLE
sleap-label command does not launch GUI properly on macOS

### DISCUSSION THREAD

**Discussion by Indraniiii** [talmolab/sleap#1320] [Help!]
Hi, 

I am trying to use the sleep-label GUI for my lab research. I am currently using it on my Mac machine. When I  use the `sleep-label` command on my terminal, it gives me the following UI. 

<img width="1101" alt="Screen Shot 2023-04-22 at 2 53 10 PM" src="https://user-images.githubusercontent.com/131559385/233808241-47f51e24-b9b4-4c3d-9602-bc8fc155210c.png">

I am not sure how to navigate from here. Please can you help me with this? 
Thanks in advance! :)

**Comment by roomrys** [comment]
Hi @Indraniiii,
<img width="674" alt="Screenshot 2023-04-24 at 2 13 24 PM" src="https://user-images.githubusercontent.com/38435167/234119293-adc37381-ef11-4426-adb1-4c55e0ad5d93.png">

The file menu should show above that window where the screenshot is cut-off. If you are in full screen mode on your Mac, then hovering near the top of the screen should entice the file menu into appearing.

Let us know if you are still having problems.

Thanks,
Liezl
